addons/placement/functions/fnc_updatePreview.sqf

```diff
@@ -0,0 +1,62 @@
+#include "script_component.hpp"
+
+private _pos = AGLtoASL screenToWorld getMousePosition;
+private _class = call FUNC(getSelectedClass);
+
+#define OFFSET 10
+
+if (_class isEqualTo "") exitWith {
+    if !(GVAR(previewVehicle) isEqualTo objNull) then {
+        deleteVehicle GVAR(previewVehicle);
+        deleteVehicle GVAR(previewObject);
+        GVAR(previewVehicle) = objNull;
+    };
+};
+
+// Delete the old preview if the player selects a different item
+if !(_class isEqualTo (typeOf GVAR(previewVehicle))) then {
+    deleteVehicle GVAR(previewVehicle);
+    deleteVehicle GVAR(previewObject);
+    GVAR(previewVehicle) = objNull;
+};
+
+// Create the preview if it doesn't exist
+if (isNull GVAR(previewVehicle)) then {
+    GVAR(previewObject) = "Land_Wrench_F" createVehicleLocal (ASLtoAGL _pos);
+    GVAR(previewObject) hideObject true;
+    GVAR(previewVehicle) = _class createVehicleLocal [0, 0, 0];
+    GVAR(previewVehicle) attachTo [GVAR(previewObject), [0, 0, OFFSET + 1 + ((boundingCenter GVAR(previewVehicle)) select 2)]];
+    GVAR(previewVehicle) enableSimulationGlobal false;
+    GVAR(previewVehicle) disableCollisionWith player;
```

Comment:
Wouldn't this only disable the collision with the Zeus?

Reply:
Yes, I only had an issue with it colliding with the local player